### ISSUE TITLE
Trouble with install

### ISSUE THREAD

**Issue by lomfs24** [floft/BellSystem#4]
I am having a problem with the install. I can't really tell whether it's running or not. I followed the instructions. But when I try to hit the IP address it says that connection is refused. Everything went well with the install except php7.0-xml could not be found in the repository. I installed php7.1-xml, php7.2-xml and php7.3-xml and nothing seems to be working. I am not sure how much of a difference the php install would make. When I look at 'top' I don't see anything indicating that floft or Bell Systems are running.

**Comment by lomfs24** [comment]
I don't know what happened. I uninstalled it all and re-installed it and it seems to be working now. Who knows what happened.

**Comment by floft** [comment]
Hmm, interesting. Glad it started working!

**Comment by lomfs24** [comment]
Hey Garrett,
Thanks for the reply. I think it was an ID10T error. I had the pi being
used for other things. Then I stripped it down and reloaded raspbian
Buster. I think it was working correctly all along. But my browser was
still looking for the IP address at https:// of course the school bell
system is not a secured website so it is at http:// Once I cleared the
cache on my browser it started working fine.

Rahn
